### PR TITLE
Two easy box tweaks

### PR DESCRIPTION
**Pull request by Probe1**
Tweaks Runtime to his rightful CMO spot
Tweaks Interrogation room doors to be named so they reflect the name of the area

Closes https://github.com/d3athrow/vgstation13/issues/10218
Closes https://github.com/d3athrow/vgstation13/issues/10139

### CODE REVIEW

**Comment by Probe1** [comment]
Theres some discussion to be had in @9600bauds wanted the security room to be called processing.  Thats ok.  But there's an area already pathed as processing (but named prisoner education something) and there'd have to be a shuffle and map checks to see if its used anywhere.  I'm willing to do it if people want but I'm not really keen on a buncha work

**Comment by Shadowmech88** [comment]
I don't care if it's called Processing specifically, but I don't think it should be called Interrogation Room given that there's already an Interrogation room.

File: maps/tgstation.dmm [hunk 1]
```diff
@@ -4029,7 +4029,7 @@
 "bzy" = (/obj/structure/disposalpipe/segment,/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden{dir = 4},/obj/machinery/atmospherics/pipe/simple/supply/hidden,/turf/simulated/floor{dir = 4; icon_state = "whiteblue"},/area/medical/medbay)
 "bzz" = (/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden{dir = 4},/obj/machinery/door/firedoor/border_only{dir = 8; name = "Firelock West"},/obj/machinery/door/airlock/glass_medical{id_tag = ""; name = "Paramedic Station"; req_access_txt = "500"},/turf/simulated/floor{icon_state = "white"},/area/medical/paramedics)
 "bzA" = (/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden{dir = 4},/turf/simulated/floor{icon_state = "white"},/area/medical/paramedics)
-"bzB" = (/obj/machinery/atmospherics/unary/vent_scrubber/on{dir = 8},/turf/simulated/floor{icon_state = "white"},/area/medical/paramedics)
+"bzB" = (/obj/machinery/atmospherics/unary/vent_scrubber/on{dir = 8},/mob/living/simple_animal/cat/snek/corpus,/turf/simulated/floor{icon_state = "white"},/area/medical/paramedics)
```
Comment (ComicIronic) [inline maps/tgstation.dmm:4032]: > cat/snek

This is a path that should be reserved for a ferret.
Also, what?

Comment (Probe1) [inline maps/tgstation.dmm:4032]: I saw that and shook my head as well. 

Why the fuck

Comment (9600bauds) [inline maps/tgstation.dmm:4032]: They both hunt mice etc
They have literally no distinct functionality, so why not?
bear/skrite was ACTUALLY bad

Comment (NigglyWiggly) [inline maps/tgstation.dmm:4032]: I-it was just so I could have the snakes hunt mice like cats, and share similar behaviors, I swear.

**Pull request merged**